ups/eupspkg.cfg.sh

```diff
@@ -1,6 +1,6 @@
 # EupsPkg config file. Sourced by 'eupspkg'
 
-MINICONDA2_VERSION=${MINICONDA2_VERSION:-3.19.0} # Version of Miniconda to install
+MINICONDA3_VERSION=${MINICONDA3_VERSION:-4.0.5} # Version of Miniconda to install
```

Comment:
Is there a reason you are using 4.0.5? The current version is 4.2.1. See: https://github.com/conda/conda/blob/master/CHANGELOG.md